### DISCUSSION TITLE
OsmAnd+ on multiple devices

### DISCUSSION THREAD

**Discussion by WB0OCV** [osmandapp/OsmAnd#14008] [Q&A]
I have a paid copy of OsmAnd+, version 4.1.11, installed on my Google Pixel 2 smartphone.  I also have the free version (4.1.11) installed on an older Samsung tablet.
Can I install my existing paid version of OsmAnd+ on the Samsung tablet, without having to purchase an additional 'license', or do I need to purchase another 'license' for another paid copy of the program?  Thank you.

**Comment by mikehgentry** [comment]
You can install it on as many devices with the Google Play store as you own, as far as I'm aware. I've certainly done it on several simultaneously without it complaining.

You can also get Osmand~ for free from FDroid, which is essentially the same as Osmand+, if you need to install it somewhere without the Play Store.

**Reply by mwerle** [reply]
Hmm, I literally just bought a phone to use as a GPS unit for my motorbike with the express purpose of using my copy of OsmAnd+ on it - and it didn't automatically install on my new phone.  Checking Google Play Store, it's asking me to buy it again.

**Reply by DmitryAlexei** [reply]
You may have changed your account in the Play Store (need to be installed from the same one). 
Also possible, that you have purchased the + version inside the free app. In that case, you need to install the free version and restore purchases.

**Reply by mwerle** [reply]
Same play store account; and "restore purchases" does nothing.  I just created an account for OsmAnd cloud (I've never used it, perhaps it didn't even exist back when I first got OsmAnd) and it, unsurprisingly, also says there's no subscription.

Hrmn.

**Reply by DmitryAlexei** [reply]
@mwerle You'd better contact the customer support on your issue (support@osmand.net)
